app-nuxtjs-starter/pages/index.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+      <Layout>
+        <p v-if="$fetchState.pending">Fetching mountains...</p>
+        <p v-else-if="$fetchState.error">An error occurred :(</p>
+        <div v-else>
+          <p>
+            Hello {{userName}} 👋
+          </p>
+          <p>
+            There are the first {{stories.length}} stories on the {{props.spaceName}} space:
+          </p>
+          <Stories stories={stories} />
+        </div>
+    </Layout>
+</template>
+
+<script>
+import {isAppSessionQuery, sessionCookieStore} from "@storyblok/app-extension-auth";
+import {authHandlerParams, endpointPrefix} from "../auth";
+
+export default {
+  data() {
+    return {
+      userName  :undefined,
+      stories   :undefined,
+      spaceName :undefined,
+      spaceId   :undefined,
+      userId    :undefined,
+    }
+  },
+
+  methods: {
+    fetchStories(spaceId, userId){
+      fetch(`api/stories?spaceId=${spaceId}&userId=${userId}`)
+        .then(res => res.json())
+        .then(stories => stories)
+        .catch(error => {
+          console.error(error)
+          return []
+        })
+      }
+  },
+
+  async fetch(context) {
```

Comment:
As we discussed. The app-extension-auth package is a nodejs (server-side) package